modules/mobile/app/workers/mobile/v0/vaccines_updater_job.rb

```diff
@@ -41,27 +40,26 @@ def perform
 
       def aggregate_source_data(aggregate, node)
         cvx_code = find_value(node, 'CVXCode')
-        group_name = find_value(node, 'Vaccine Group Name')
+        manufacturer = find_value(node, 'Manufacturer')
+
+        return nil if cvx_code.blank? || manufacturer.blank?
```

Comment:
there's some expected blank manufacturers but looks like on the manufacturers xml the cvx code is the primary unique identifier and doesn't have any blanks so I'll add an error for that.

Reply:
Thinking about this more... if we add logging or error handling, it'll happen every day, since the source data is mostly static. I'd run this locally first and confirm that all nodes have cvx codes. If they do, then we could add some error logging for the future. If not, maybe we just consider it normal behavior and keep the code as it is.

Reply:
I keep thinking about this. Honestly, I'm not sure we care. It's the CDC's data. It's really not up to us to verify their data integrity. If a record doesn't have all the info, skipping seems reasonable. Probably best to just leave it as is.

Reply:
That's fair. Could be worse off if the sentry error flies under the radar and we kill the job over again and never get any valid new vaccine record unknowingly just due to a missing cvx code of one of them.